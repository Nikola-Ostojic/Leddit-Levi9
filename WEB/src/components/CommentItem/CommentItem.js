import React from "react";
import { Card } from "@material-ui/core";

import styles from "./CommentItem.module.css";

const CommentItem = ({ comment }) => (
  <Card className={styles.container}>
    <div className={styles.headerContainer}>
      <p>{comment.content}</p>
      <p>{comment.author}</p>
    </div>
  </Card>
);

export default CommentItem;
