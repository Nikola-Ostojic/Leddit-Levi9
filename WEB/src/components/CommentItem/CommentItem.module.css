.container {
    margin-top: 35px;
    padding: 15px;
    box-sizing: border-box;
    border-left: 6px solid darkblue;
}

.headerContainer {
    width: 100%;
    display: flex;
    justify-content: space-between;
    color: darkslateblue;
    background-color: whitesmoke;
}