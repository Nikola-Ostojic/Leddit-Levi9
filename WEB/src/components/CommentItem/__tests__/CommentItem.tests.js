import React from "react";
import { shallow } from "enzyme";

import CommentItem from "../CommentItem";

describe("CommentItem component tests", () => {
  const mockedProps = {
    comment: {
      content: "Test"
    }
  };
  it("Should match snapshot", () => {
    const wrapper = shallow(<CommentItem {...mockedProps} />);

    expect(wrapper).toMatchSnapshot();
  });
});
