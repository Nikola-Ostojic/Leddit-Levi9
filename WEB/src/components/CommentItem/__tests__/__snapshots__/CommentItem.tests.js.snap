// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CommentItem component tests Should match snapshot 1`] = `
<WithStyles(ForwardRef(Card))
  className="container"
>
  <div
    className="headerContainer"
  >
    <p>
      Test
    </p>
    <p />
  </div>
</WithStyles(ForwardRef(Card))>
`;
